src/diceBot/CodeLayerd.rb

```diff
@@ -31,48 +40,57 @@ def rollDiceCommand(command)
     result = ''
 
     case command
-    when /(\d+)?CL(\@?(\d))?(>=(\d+))?/i
-      base = (Regexp.last_match(1) || 1).to_i
-      target = (Regexp.last_match(3) || 6).to_i
-      diff = Regexp.last_match(5).to_i
-      result = checkRoll(base, target, diff)
+    when /(\d+)?CL([+-]\d+)?(\@(\d))?(\[(\d{1,2})\])?([+-]\d+)?(>=(\d+))?/i
+      m = Regexp.last_match
+      base = (m[1] || 1).to_i
+      modifier1 = (m[2].to_i || 0)
```

Comment:
```suggestion
      modifier1 = m[2].to_i
```

`nil.to_i` は0になるので、 `||`とらなくて大丈夫です